bin/run.py

```diff
@@ -319,28 +315,18 @@ def process_run(run, p):
 
             localbar.done(got_expected, f'{result.duration:6.3f}s {result.print_verdict()}', data)
 
-            # Lazy judging: stop on the first error when:
-            # - not in verbose mode
-            if config.args.verbose or config.args.table:
-                return
-            # - the result has max priority
-            if result.verdict not in config.MAX_PRIORITY_VERDICT:
-                return
             # - for TLE, the run was aborted because the global timeout expired
-            if result.verdict == 'TIME_LIMIT_EXCEEDED' and not result.timeout_expired:
+            # TODO: What does this mean?
+            if result.verdict == Verdict.TIME_LIMIT_EXCEEDED and not result.timeout_expired:
                 return
 
-            bar.count = None
-            p.abort()
-
         p = parallel.new_queue(lambda run: process_run(run, p), pin=True)
         for run in runs:
             p.put(run)
         p.done()
 
-        self.verdict = verdict[1]
-        self.print_verdict = verdict[2]
-        self.duration = max_duration
+        self.verdict = verdicts['.']
+        self.duration = 42  # TODO this is messed up
```

Comment:
This can probably be removed? :stuck_out_tongue: I don't see the `42` ending up in my console so I guess that `self.duration` not needed? :joy: 

(it's nice that the `duration` bookkeeping is now part of `verdicts`, btw :smile:)